src/edgepi/eeprom/proto_files/edgepi_module.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+import "dac_module.proto";
+import "adc_module.proto";
+import "rtd_module.proto";
+import "tc_module.proto";
+import "keys.proto";
+
+message EepromData {
+    optional DacModule dac_module = 1;
+    optional AdcModule adc_module = 2;
```

Comment:
I would change `AdcModule` message to only have channels and diffs defined like this (no mention of adc):
```
message AdcModule {
    optional calib_param ch_1 = 1;  
    optional calib_param ch_2 = 2; 
    optional calib_param ch_3 = 3;  
    optional calib_param ch_4 = 4;  
    optional calib_param ch_5 = 5;  
    optional calib_param ch_6 = 6;  
    optional calib_param ch_7 = 7;  
    optional calib_param ch_8 = 8;  

    optional calib_param diff_1 = 9;  
    optional calib_param diff_2 = 10;  
    optional calib_param diff_3 = 11;  
    optional calib_param diff_4 = 12;  

```
then here:
```
    optional AdcModule adc1_module = 2;
    optional AdcModule adc2_module = 3;
```